flocker_bb/builders/maint.py

```diff
@@ -75,6 +75,26 @@ def fmap(_f, _value, *args, **kwargs):
     else:
         return _f(_value, *args, **kwargs)
 
+# FLOC-2288
+# New CleanVolumes LoggingBuildStep
+# parameterized on age (lag, like CleanAcceptanceInstances)
+#
+# libcloud configured just as it is in CleanAcceptanceInstances
```

Comment:
One thing to note is that we will create volumes in two regions on AWS. One for the functional tests and one for the acceptance tests.